JavaToPython/Grammers/Java.Grm

```diff
@@ -71,7 +71,12 @@ end define
 
 define class_name
     [id]'< [list id] '>
-    | [id]   
+    | [id] [repeat array_declaration]
```

Comment:
Why repeat here?
Also "Person[]" is not a class name, just the "Person" part is so we should create a new type for arrays entirely

Reply:
> Why repeat here?

Because Person[][] could be a thing

Good point on the grammar though... I tested it quickly and it seemed to work, but it might need more thinking logic wise. (not that much though).